zipkin-storage/elasticsearch/src/test/java/zipkin2/elasticsearch/integration/OpenSearchExtension.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package zipkin2.elasticsearch.integration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import static org.testcontainers.utility.DockerImageName.parse;
+
+class OpenSearchExtension extends ElasticsearchBaseExtension {
+  static final Logger LOGGER = LoggerFactory.getLogger(OpenSearchExtension.class);
+
+  OpenSearchExtension(int majorVersion) {
+    super(new OpenSearchContainer(majorVersion));
+  }
+
+  // mostly waiting for https://github.com/testcontainers/testcontainers-java/issues/3537
+  static final class OpenSearchContainer extends GenericContainer<OpenSearchContainer> {
+      OpenSearchContainer(int majorVersion) {
+      super(parse("ghcr.io/openzipkin/zipkin-opensearch" + majorVersion + ":3.3.1"));
```

Comment:
@codefromthecrypt could you clarify a thing for me here please: I prepared this image for testing in this pull request but I need it published before, how it usually works? thanks!

Reply:
(PS: I've been using the image locally for testing)